src/main/java/types/ClusterUpgradeAction.java

```diff
@@ -37,6 +37,18 @@
      * @status added
      */
     START,
+
+    /**
+     * The upgrade action to be passed to update the cluster upgrade progress. This should be used
+     * as the upgrade progresses.
+     *
+     * @author Scott Dickerson <sdickers@redhat.com>
+     * @date 16 Mar 2022
+     * @since 4.5.0
+     * @status added
+     */
+    UPDATE,
```

Comment:
the word 'update' is generic, I think UPDATE_PROGRESS would be more informative:  and then the numeric input can be named 'progressPercent'